[2023-12-26T09:28:36.557+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_filtered_customers_table manual__2023-12-26T09:28:32.425780+00:00 [queued]>
[2023-12-26T09:28:36.568+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_filtered_customers_table manual__2023-12-26T09:28:32.425780+00:00 [queued]>
[2023-12-26T09:28:36.569+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-26T09:28:36.569+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-12-26T09:28:36.569+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-26T09:28:36.580+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_filtered_customers_table> on 2023-12-26 09:28:32.425780+00:00
[2023-12-26T09:28:36.588+0000] {standard_task_runner.py:55} INFO - Started process 4097 to run task
[2023-12-26T09:28:36.593+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'potential_customers_process', 'create_filtered_customers_table', 'manual__2023-12-26T09:28:32.425780+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/potential_customers_process.py', '--cfg-path', '/tmp/tmppmd6r47b']
[2023-12-26T09:28:36.596+0000] {standard_task_runner.py:83} INFO - Job 17: Subtask create_filtered_customers_table
[2023-12-26T09:28:36.677+0000] {task_command.py:376} INFO - Running <TaskInstance: potential_customers_process.create_filtered_customers_table manual__2023-12-26T09:28:32.425780+00:00 [running]> on host 992e73c39b9a
[2023-12-26T09:28:36.750+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=potential_customers_process
AIRFLOW_CTX_TASK_ID=create_filtered_customers_table
AIRFLOW_CTX_EXECUTION_DATE=2023-12-26T09:28:32.425780+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-26T09:28:32.425780+00:00
[2023-12-26T09:28:36.760+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-12-26T09:28:36.764+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS filtered_customers (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                gender TEXT NOT NULL,
                age INT NOT NULL,
                last_login TEXT NOT NULL,
                external_id TEXT NOT NULL
            );
        , parameters: None
[2023-12-26T09:28:36.765+0000] {postgres.py:96} INFO - NOTICE:  relation "filtered_customers" already exists, skipping

[2023-12-26T09:28:36.773+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=potential_customers_process, task_id=create_filtered_customers_table, execution_date=20231226T092832, start_date=20231226T092836, end_date=20231226T092836
[2023-12-26T09:28:36.806+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-26T09:28:36.834+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
